net/third_party/quiche/src/quiche/quic/core/quic_constants.h

```diff
@@ -294,8 +294,8 @@ inline constexpr int kDefaultIetfLossDelayShift = 3;
 
 // Maximum number of retransmittable packets received before sending an ack.
 inline constexpr QuicPacketCount kDefaultRetransmittablePacketsBeforeAck = 2;
-// Wait for up to 10 retransmittable packets before sending an ack.
-inline constexpr QuicPacketCount kMaxRetransmittablePacketsBeforeAck = 10;
+// Wait for up to 50 retransmittable packets before sending an ack.
+inline constexpr QuicPacketCount kMaxRetransmittablePacketsBeforeAck = 50;
```

Comment:
Wrap this in an `#if defined(USE_COBALT_CUSTOMIZATIONS)`

Reply:
Done